tests/integration/test_mixed_space_bayesian_optimization.py

```diff
@@ -55,16 +52,16 @@
             ]
         ],
         [
-            (15, EfficientGlobalOptimization()),
+            (24, EfficientGlobalOptimization()),
```

Comment:
How are the test run times affected by this PR (both alltests and quicktests)?

Reply:
some increase, but not that much, seemed worthwhile the simplification of the code

Reply:
it seems like here running time is almost not affected at all, 5s more or less